addons/web/static/src/views/fields/many2many_tags/many2many_tags_field.scss

```diff
@@ -45,9 +45,7 @@
         }
 
         .o_badge_text, .o_tag_badge_text {
-            @include o-text-overflow(inline-block);
-            max-width: 200px;
-            color: inherit;
+            @include o-text-overflow;
             line-height: 1.1;
```

Comment:
I'm not sure I get the relation between those changes and the fix... :thinking:

Reply:
Also, the branch is :red_circle:  once rebuilt (and should probably be rebased)(both may be linked, btw)

Reply:
Indeed, its seems to work without. @adr-odoo Do you remember why that was there ?

Reply:
@pparidans @adr-odoo  I only kept the changes in `dashboard_action.scss`. I'm assuming it's Ok like this :shrug: